lib/game_status.ex

```diff
@@ -0,0 +1,30 @@
+defmodule TicTacToe.GameStatus do
+  @win_combos [
+    {1, 2, 3}, 
+    {4, 5, 6},
+    {7, 8, 9},
+    {1, 4, 7},
+    {2, 5, 8},
+    {3, 6, 9},
+    {1, 5, 9},
+    {3, 5, 7}
+  ]
+
+  def check_win(board) do
+    winner = case @win_combos |> Enum.find(fn {pos1, pos2, pos3} ->
```

Comment:
Everything in the case should be pulled out into a separate func IMO, or is there a way you can pull out `board[winner]` into `winner` so this is cleaner?

Reply:
Extracted the case statement into a separate winner function that just returns the result of the case.